test/run-drun/ok/actor-class-mgmt.drun-run.ok

```diff
@@ -1,3 +1,4 @@
 ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
 ingress Completed: Reply: 0x4449444c0000
+debug.print: {controllers = [rwlgt-iiaaa-aaaaa-aaaaa-cai]; recent_changes = [{canister_version = 0; details = #creation({controllers = [rwlgt-iiaaa-aaaaa-aaaaa-cai]}); origin = #from_canister({canister_id = rwlgt-iiaaa-aaaaa-aaaaa-cai; canister_version = null})}, {canister_version = 1; details = #code_deployment({mode = #install}); origin = #from_canister({canister_id = rwlgt-iiaaa-aaaaa-aaaaa-cai; canister_version = ?18})}, {canister_version = 3; details = #code_deployment({mode = #upgrade}); origin = #from_canister({canister_id = rwlgt-iiaaa-aaaaa-aaaaa-cai; canister_version = ?22})}, {canister_version = 5; details = #code_deployment({mode = #reinstall}); origin = #from_canister({canister_id = rwlgt-iiaaa-aaaaa-aaaaa-cai; canister_version = ?26})}]; total_num_changes = 4}
```

Comment:
`canister_version = null` is a bit fishy. Where is the original creation code?

Reply:
Is it in line 74?
``` Motoko
let c0 = await
        Cs.C (...
```
?

Reply:
I guess it is rather https://github.com/dfinity/motoko/blob/39d64259f998a993d387afce8c2ee0dc0741c9e3/test/run-drun/actor-class-mgmt.mo#LL92C20-L92C20

Reply:
Indeed, I tried
``` diff
$ git diff
diff --git a/test/run-drun/actor-class-mgmt.mo b/test/run-drun/actor-class-mgmt.mo
index b229bcfc6..ea4d6f8fb 100644
--- a/test/run-drun/actor-class-mgmt.mo
+++ b/test/run-drun/actor-class-mgmt.mo
@@ -35,6 +35,7 @@ actor a {
   let ic00 = actor "aaaaa-aa" :
     actor {
       create_canister : {
+          sender_canister_version : ?Nat64;
         settings : ? {
           controllers : ?[Principal];
           compute_allocation: ?Nat;
@@ -89,7 +90,7 @@ actor a {
 
       Cycles.add(2_000_000_000_000);
       let {canister_id = p} = await
-         ic00.create_canister default_settings;
+        ic00.create_canister { settings = null; sender_canister_version = ?(Prim.canisterVersion()) };
       // no need to add cycles
       let c3 = await
         (system Cs.C)(#install p)(3, null);
```
and found that field filled in.